nni/experiment/experiment.py

```diff
@@ -136,7 +132,7 @@ def stop(self) -> None:
         _logger.info('Stopping experiment, please wait...')
         atexit.unregister(self.stop)
 
-        nni.runtime.log.stop_experiment_log(self.id)
+        stop_experiment_logging(self.id)
```

Comment:
What if I resume the experiment after stop?

I think either we have an API to explicitly re-enable it, or we postpone this stop to `__del__`.

Reply:
`resume` also use `start()` to resume the experiment, so it is ok?